core/ir/instr_inline_api.h

```diff
@@ -141,6 +141,13 @@ opnd_is_far_base_disp(opnd_t op)
     return IF_X86_ELSE(op.kind == BASE_DISP_kind && op.aux.segment != DR_REG_NULL, false);
 }
 
+INSTR_INLINE
+bool
+opnd_is_element_vector_reg(opnd_t op)
+{
+    return op.kind == REG_kind && ((op.aux.flags & DR_OPND_IS_VECTOR) != 0);
```

Comment:
style: Use TEST

Reply:
TEST isn't actually available in this file and adding the header file it appears in causes quite a lot of problems. 

I'm surprised the TEST at the top of the file doesn't cause issues too